client/me/help/help-contact-form/index.jsx

```diff
@@ -225,12 +226,36 @@ export class HelpContactForm extends React.PureComponent {
 		return !! message.trim();
 	};
 
+	/**
+	 * Determine if the additional form is ready to submit
+	 * @return {bool} Return true if the additional support option can be used
+	 */
+	canSubmitAdditionalForm = () => {
+		const { disabled } = this.props;
+		const { subject, message } = this.state;
+
+		if ( disabled ) {
+			return false;
+		}
+
+		if ( ! subject.trim() ) {
+			return false;
+		}
+
+		return !! message.trim();
+	};
+
 	/**
 	 * Start a chat using the info set in state
 	 * @param  {object} event Event object
 	 */
 	submitForm = () => {
 		const { howCanWeHelp, howYouFeel, message, subject } = this.state;
+		const { additionalSupportOption } = this.props;
+
+		if ( additionalSupportOption && additionalSupportOption.enabled ) {
+			this.props.recordTracksEvent( 'calypso_happychat_a_b_english_chat_selected' );
```

Comment:
For this, and the other Tracks events, can you send additional props to note the customer's locale? That lets us also identify if there's a strong difference between `es` and `pt-br` speakers.

```
this.props.recordTracksEvent( 'calypso_happychat_a_b_english_chat_selected', { locale: customerLanguage } );
```

Reply:
Great suggestion, added.